app/views/layouts/application.html.erb

```diff
@@ -7,19 +7,35 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <div class="container">
-    <div class="page-header">
-      <h1>Welcome to Tutorify</h1>
-    </div>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="#">Tutorify</a>
+      </div>
 
-    <div class="row">
-      <% if signed_in? %>
-        <div class="pull-right">
-          <p>Welcome <%= current_user.name %>!</p>
-          <%= link_to 'Sign out', session_path, method: :delete %>
-        </div>
-      <% end %>
-    </div>
+      <!-- Collect the nav links, forms, and other content for toggling -->
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```

Comment:
you have assigned an `id` here but I don't see any specific styles for that id anywhere.